spec/oncocore_vs.txt

```diff
@@ -124,16 +121,16 @@ Includes codes descending from SCT#254293002 "Tumor-node-metastasis (TNM) tumor
   SCT#258235000 "International Union Against Cancer (tumor staging)"
 
 ValueSet: TNMStageGroupVS
-Description: "Classification of the extent of the disease, according to the TNM staging rules, typically taking into account several anatomic and prognostic elements."
+Description: "Category describing the extent of the disease, according to the TNM staging rules, typically taking into account several anatomic and prognostic elements."
```

Comment:
Should be "Categories". Each item in the value set is a category.

Reply:
Done.